3001-4000/LIT3592Miracle.xml

```diff
@@ -92,81 +112,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                            <desc>
-                               <ref type="mss" corresp="EMML6640">EMML 6640, f. 19r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2059">
-                            <desc>
-                               <ref type="mss" corresp="EMML2059">EMML 2059, f. 135r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2060">
-                            <desc>
-                               <ref type="mss" corresp="EMML2060">EMML 2060, f. 183r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2066">
-                            <desc>
-                               <ref type="mss" corresp="EMML2066">EMML 2066, f.  60r, 115v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2802">
-                            <desc>
-                               <ref type="mss" corresp="EMML2802">EMML 2803, f. 66v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML6196">
-                            <desc>
-                               <ref type="mss" corresp="EMML6196">EMML 6196, f. 19v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML7543">
-                            <desc>
-                               <ref type="mss" corresp="EMML7543">EMML 7543, f. 27r</ref>
-                            </desc>
-                         </relation>
+
             </listRelation>
          </div>
          <div type="edition">
-             <div type="textpart" subtype="incipit" xml:lang="gez">
-                <note>This incipit is taken from <ref type="mss" corresp="EMML5520"/>.</note>
-                 <ab>
-                     ወኮነ፡ በዝንቱ፡ ደብር፡ ቅዱስ፡ ዘዝኩር፡ በመዋዕሊሁ፡ ለአቡነ፡ አባ፡ ገብርኤል፡ ሊቀ፡ ጳጳሳት፡ ዘለእስክንድርያ፡
-                     ብእሲ፡ መነኮስ፡ ቅዱስ፡ ወእምንእሱ፡ ነበረ፡ ህየ፡ ወስሙ፡ ይስሐቅ።
-                 </ab>
-             </div>
             <div type="textpart" subtype="incipit" xml:lang="gez">
-               <note>This incipit is taken from <ref type="mss" corresp="EMML4205"/>.</note>
-               <ab>
-                  ወሀሎ፡ አሐዱ፡ መነኮስ፡ ኄር፡ ዘስሙ፡ ይስሐቅ፡ ወያፈቅራ፡ ለእግዝእትነ፡ ማርያም፡
-                  ወእምንእሱ፡ ነበረ፡ በአሐቲ፡ ደብር፡ በመዋዕሊሁ፡ ለአባ፡ ገብርኤል።
-               </ab>
+               <note>This incipit, related to the first miracle version is taken from <ref
```

Comment:
I think, these incipits should go the newly created CAe to which they belong. If this is a generic record, it should not contain any incipit or explicit or any other text. Otherwise the encoders and users will be confused about the correct CAe.